src/components/Input/index.tsx

```diff
@@ -61,6 +67,12 @@ const Input: React.FC<Props> = props => {
   const descriptionClass = 'input--description';
   const labelClass = 'input--label';
 
+  const handleBlur = event => {
+    onBlur
+      ? onBlur()
```

Comment:
here we should call both the on blur and the set validation, not one or the other.